src/site/apt/index.apt.vm

```diff
@@ -30,7 +30,7 @@
 ${project.name}
 
   The Checkstyle Plugin generates a report regarding the code style used by the developers. For more information about Checkstyle,
-  see {{{http://checkstyle.sourceforge.net/}http://checkstyle.sourceforge.net/}}.
+  see {{{https://checkstyle.sourceforge.io/}https://checkstyle.sourceforge.io/}}.
```

Comment:
Why don't you use https://checkstyle.org/ ?

Reply:
We migrated all our documentation to `org` website, only old website versions left on sourceforge.

sourceforge will stay till https://github.com/checkstyle/checkstyle/issues/7377 , after this issue be resolved we we will setup full redirection of sourceforge to checkstyle.org .

Reply:
done